Description: No dependency on Windows
Author: John Goerzen <jgoerzen@complete.org>
Last-Update: 2023-05-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: b/Cargo.toml
===================================================================
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,3 +45,2 @@ close = [
     "hermit-abi",
-    "windows-sys",
 ]
@@ -88,12 +87 @@ optional = true
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.48.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-    "Win32_Networking_WinSock",
-    "Win32_Security",
-    "Win32_System_IO",
-    "Win32_System_Threading",
-]
-optional = true
